src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchProfileExtensions.cs

```diff
@@ -67,5 +70,18 @@ public static bool IsRemoteDebugEnabled(this ILaunchProfile profile)
 
             return null;
         }
+
+        public static Guid RemoteAuthenticationPortSupplier(this ILaunchProfile profile)
```

Comment:
I would consider that we persist friendly names here, but respect a GUID if specified. Persisting guids in file format isn't the most friendly of things.

Reply:
I thought about that, but I thought the idea that launchSettings.json was valid independent of mapping that VS provides might be useful for other potential consumers, like Code or VS Mac etc.

Reply:
Help me undersatnd how GUIDs provide that that strings don't? These GUIDs are tied to COM objects that live in VS.

Reply:
Is that what they are? In that case, I agree, but I couldn't find many references to them (other than passing them to the debugger) so I assumed they meant something important.